packages/page-claims/src/Claim.tsx

```diff
@@ -26,12 +26,16 @@ interface Props {
 
 // Depending on isOldClaimProcess, construct the correct tx.
 function constructTx (accountId: string, ethereumSignature: string | null, kind: StatementKind | undefined, isOldClaimProcess: boolean): {
-  params: any[];
-  tx: string;
+  params?: any[];
+  tx?: string;
```

Comment:
If this type is big enough to split over multiple lines (severely decreasing readability), extract it into an interface.